daemon/mgr/spec_volume.go

```diff
@@ -14,15 +14,20 @@ func setupMounts(ctx context.Context, c *ContainerMeta, s *specs.Spec) error {
 		return nil
 	}
 	for _, v := range c.HostConfig.Binds {
-		sd := strings.SplitN(v, ":", 2)
-		if len(sd) != 2 {
+		sd := strings.Split(v, ":")
+		lensd := len(sd)
+		if lensd < 2 || lensd > 3 {
```

Comment:
We can touch a name which named a`x:y`, so "a:x:y:rw" may be OK, so we cannot be so strict.